### DISCUSSION TITLE
list all key of bucket in memory mode

### DISCUSSION THREAD

**Discussion by hnimminh** [nutsdb/nutsdb#199] [Q&A] [answered]
@xujiajun 
Is there  a way to list all key of bucket in memory mode?
Example:
I have 2x key:value in BUCKET_KV, How to list all key?

**Comment by xujiajun** [comment, marked as answer]
@hnimminh The memory model has no iteration api, only a range api, you can submit an issue to support.

**Reply by xujiajun** [reply]
I submitted an issue for you  https://github.com/nutsdb/nutsdb/issues/200

**Reply by hnimminh** [reply]
Thanks @xujiajun, currently I use nutsdb replace for redis as embeded database, where redis can not fill this role. Basically I'm happy with nustdb, and wish it has more like redis capacity. anyway thanks for your great works!
